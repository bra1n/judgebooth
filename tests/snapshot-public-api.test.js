const instance = require('./utils/client');

test('snapshot /?action=sets', async () => {
    const response = await instance.get('/?action=sets');
    expect(response.data).toMatchSnapshot();
});

test('snapshot /?action=questions', async () => {
    const response = await instance.get('/?action=questions');
    expect(response.data).toMatchSnapshot();
});

test('snapshot /?action=question&id=-2', async () => {
    // TODO: probably we should return 404
    const response = await instance.get('/?action=question&id=-2');
    expect(response.data).toMatchSnapshot();
});

test('snapshot /?action=question&id=58', async () => {
    const response = await instance.get('/?action=question&id=58');
    expect(response.data).toMatchSnapshot();
});

test('snapshot /?action=question&id=58&lang=1', async () => {
    const response = await instance.get('/?action=question&id=58&lang=1');
    expect(response.data).toMatchSnapshot();
});

test('snapshot /?action=offline', async () => {
    const response = await instance.get('/?action=offline');
    expect(response.data).toMatchSnapshot();
});