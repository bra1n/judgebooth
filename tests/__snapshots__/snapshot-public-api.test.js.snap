// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot /?action=offline 1`] = `
Object {
  "cards": Object {
    "2827": Object {
      "id": "2827",
      "layout": "normal",
      "manacost": "{2}{W}{W}",
      "multiverseid": "397614",
      "name": "Humility",
      "text": "All creatures lose all abilities and have base power and toughness 1/1.",
      "type": "Enchantment",
    },
    "3384": Object {
      "id": "3384",
      "layout": "normal",
      "manacost": "{3}{R}",
      "multiverseid": "397575",
      "name": "Pandemonium",
      "text": "Whenever a creature enters the battlefield, that creature's controller may have it deal damage equal to its power to target creature or player of his or her choice.",
      "translations": Object {
        "11": "Pandémonium",
        "12": "Pandemônio",
        "2": "Pandämonium",
        "3": "Pandemonio",
        "4": "伏魔殿",
        "6": "Pandemônio",
        "7": "Столпотворение",
        "8": "Pandemónium",
        "9": "大混乱",
      },
      "type": "Enchantment",
    },
    "4187": Object {
      "id": "4187",
      "layout": "normal",
      "manacost": "{2}{W}{W}",
      "multiverseid": "15142",
      "name": "Opalescence",
      "text": "Each other non-Aura enchantment is a creature in addition to its other types and has base power and base toughness each equal to its converted mana cost.",
      "type": "Enchantment",
    },
    "4202": Object {
      "id": "4202",
      "layout": "normal",
      "manacost": "{3}{W}",
      "multiverseid": "15143",
      "name": "Replenish",
      "text": "Return all enchantment cards from your graveyard to the battlefield. (Auras with nothing to enchant remain in your graveyard.)",
      "type": "Sorcery",
    },
    "8122": Object {
      "id": "8122",
      "layout": "normal",
      "manacost": "{2}{U}",
      "multiverseid": "83807",
      "name": "Copy Enchantment",
      "text": "You may have Copy Enchantment enter the battlefield as a copy of any enchantment on the battlefield.",
      "translations": Object {
        "11": "Copie d'enchantement",
        "12": "Cópia de Encantamento",
        "2": "Verzauberungskopie",
        "3": "Copia Incantesimo",
        "4": "エンチャント複製",
        "6": "Cópia de Encantamento",
        "7": "Копия чар",
        "8": "Encantamiento copia",
        "9": "复制结界",
      },
      "type": "Enchantment",
    },
  },
  "questions": Object {
    "58": Object {
      "1": Object {
        "answer": "0 Damage. Copy Enchantment can't copy anything that enters the battlefield at the same time as it does. Pandemonium does not have any ability, because of Humility and Opalescence.",
        "question": "With Pandemonium, Humility, Copy Enchantment, and Opalescence in the graveyard, I cast Replenish. What's the highest amount of damage that I could deal?",
      },
      "10": Object {
        "answer": "0點。複製結界不能複製與其同時進入戰場的其他結界。由於謙卑和蛋白石光輝，大混亂沒有異能。",
        "question": "我的墳墓場裡有大混亂/Pandemonium、謙卑/Humility、複製結界/Copy Enchantment和蛋白石光輝/Opalescence，我施放充盈/Replenish。請問我最多能造成多少點傷害？",
      },
      "11": Object {
        "answer": "0 blessure. La Copie d'enchantement ne peut pas copier quelque chose qui arrive sur le champ de bataille en même temps qu'elle. Pandémonium n'a aucune capacité, à cause d'Humility et Opalescence.",
        "question": "Avec Pandémonium, Humility, Copie d'enchantement et Opalescence dans le cimetière, je lance Replenish. Combien de blessures au maximum puis-je infliger ?",
      },
      "2": Object {
        "answer": "0 Schaden. Die Verzauberungskopie kann nichts kopieren, das gleichzeitig mit ihr das Schlachtfeld betritt. Pandämonium hat keine Fähigkeiten, da sie von Opalessenz und Humility betroffen ist.",
        "question": "Während ich Pandämonium, Humility, Verzauberungskopie und Opalessenz auf meinem Firedhof habe, wirke ich Replenish. Wie hoch ist die größte Anzahl an Schaden, die ich zufügen könnte?",
      },
      "4": Object {
        "answer": "０点。《エンチャント複製》は同時に戦場にでるエンチャントのコピーになることはできません。《謙虚》と《オパール色の輝き》があるので、《伏魔殿》は全ての能力を失っています。",
        "question": "《伏魔殿／Pandemonium》[TSB]、《エンチャント複製／Copy Enchantment》[RAV]、《謙虚／Humility》[TMP]、《オパール色の輝き／Opalescence》[UDS]が墓地にある状態で《補充／Replenish》[UDS]を唱えました。与えられることのできる最大のダメージは何点ですか？",
      },
      "6": Object {
        "answer": "0 de dano. Cópia de Encantamento não pode copiar nada que entra no campo de batalha ao mesmo tempo que ela. Pandemônio não tem nenhuma habilidade devido a Humility e Opalescence.",
        "question": "Com Pandemônio, Humility, Cópia de Encantamento e Opalescence no cemitério, eu conjuro Replenish. Qual é o máximo de dano que eu consigo dar?",
      },
      "7": Object {
        "answer": "0 повреждений. Копия Чар не может копировать то, что входит на поле битвы в тот же момент. У Столпотворения нет способностей из-за Opalescence и Humility.",
        "question": "У меня на кладбище Столпотворение, Humility, Копия Чар и Opalescence. Я разыгрываю Replenish. Какое максимальное количество поверждений я могу нанести?",
      },
      "8": Object {
        "answer": "0 daño. Encantamiento Copia no puede copiar nada que entre al campo de batalla al mismo tiempo. Pandemónium no tiene ninguna habilidad debido a Humildad y Opalescencia.",
        "question": "Con Pandemónium, Humildad, Encantamiento Copia y Opalescencia en el cementerio lanzo Reabastecer , ¿Cuál es la mayor cantidad de daño que podría hacer?",
      },
      "9": Object {
        "answer": "0点。复制结界不能复制与其同时进入战场的其他结界。由于谦卑和蛋白石光辉，大混乱没有异能。",
        "question": "我的坟墓场里有大混乱、谦卑、复制结界和蛋白石光辉，我施放充盈。请问我最多能造成多少点伤害？",
      },
      "cards": Array [
        "8122",
        "2827",
        "4187",
        "3384",
        "4202",
      ],
    },
  },
}
`;

exports[`snapshot /?action=question&id=-2 1`] = `Array []`;

exports[`snapshot /?action=question&id=58 1`] = `Array []`;

exports[`snapshot /?action=question&id=58&lang=1 1`] = `
Object {
  "answer": "0 Damage. Copy Enchantment can't copy anything that enters the battlefield at the same time as it does. Pandemonium does not have any ability, because of Humility and Opalescence.",
  "cards": Array [
    Object {
      "layout": "normal",
      "manacost": "{2}{U}",
      "multiverseid": 83807,
      "name": "Copy Enchantment",
      "name_en": "Copy Enchantment",
      "text": "You may have Copy Enchantment enter the battlefield as a copy of any enchantment on the battlefield.",
      "type": "Enchantment",
    },
    Object {
      "layout": "normal",
      "manacost": "{2}{W}{W}",
      "multiverseid": 397614,
      "name": "Humility",
      "name_en": "Humility",
      "text": "All creatures lose all abilities and have base power and toughness 1/1.",
      "type": "Enchantment",
    },
    Object {
      "layout": "normal",
      "manacost": "{2}{W}{W}",
      "multiverseid": 15142,
      "name": "Opalescence",
      "name_en": "Opalescence",
      "text": "Each other non-Aura enchantment is a creature in addition to its other types and has base power and base toughness each equal to its converted mana cost.",
      "type": "Enchantment",
    },
    Object {
      "layout": "normal",
      "manacost": "{3}{R}",
      "multiverseid": 397575,
      "name": "Pandemonium",
      "name_en": "Pandemonium",
      "text": "Whenever a creature enters the battlefield, that creature's controller may have it deal damage equal to its power to target creature or player of his or her choice.",
      "type": "Enchantment",
    },
    Object {
      "layout": "normal",
      "manacost": "{3}{W}",
      "multiverseid": 15143,
      "name": "Replenish",
      "name_en": "Replenish",
      "text": "Return all enchantment cards from your graveyard to the battlefield. (Auras with nothing to enchant remain in your graveyard.)",
      "type": "Sorcery",
    },
  ],
  "metadata": Object {
    "author": "",
    "difficulty": 2,
    "id": 58,
    "live": true,
    "outdated": false,
  },
  "question": "With Pandemonium, Humility, Copy Enchantment, and Opalescence in the graveyard, I cast Replenish. What's the highest amount of damage that I could deal?",
}
`;

exports[`snapshot /?action=questions 1`] = `
Array [
  Object {
    "cards": Array [
      Array [
        282,
      ],
      Array [
        7457,
      ],
      Array [
        4802,
      ],
      Array [
        4802,
      ],
      Array [
        161,
      ],
    ],
    "difficulty": 2,
    "id": 58,
    "languages": Array [
      1,
      2,
      4,
      6,
      7,
      8,
      9,
      10,
      11,
    ],
  },
]
`;

exports[`snapshot /?action=sets 1`] = `
Array [
  Object {
    "code": "KLD",
    "id": 24213,
    "modern": 1,
    "name": "Kaladesh",
    "releasedate": "2016-09-30 00:00:00",
    "standard": 1,
  },
  Object {
    "code": "EMN",
    "id": 24015,
    "modern": 1,
    "name": "Eldritch Moon",
    "releasedate": "2016-07-22 00:00:00",
    "standard": 1,
  },
  Object {
    "code": "SOI",
    "id": 23821,
    "modern": 1,
    "name": "Shadows over Innistrad",
    "releasedate": "2016-04-08 00:00:00",
    "standard": 1,
  },
  Object {
    "code": "OGW",
    "id": 23630,
    "modern": 1,
    "name": "Oath of the Gatewatch",
    "releasedate": "2016-01-22 00:00:00",
    "standard": 1,
  },
  Object {
    "code": "BFZ",
    "id": 22748,
    "modern": 1,
    "name": "Battle for Zendikar",
    "releasedate": "2015-10-02 00:00:00",
    "standard": 1,
  },
  Object {
    "code": "ORI",
    "id": 22562,
    "modern": 1,
    "name": "Magic Origins",
    "releasedate": "2015-07-17 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "DTK",
    "id": 885,
    "modern": 1,
    "name": "Dragons of Tarkir",
    "releasedate": "2015-03-27 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "FRF",
    "id": 883,
    "modern": 1,
    "name": "Fate Reforged",
    "releasedate": "2015-01-23 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "KTK",
    "id": 879,
    "modern": 1,
    "name": "Khans of Tarkir",
    "releasedate": "2014-09-26 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "M15",
    "id": 878,
    "modern": 1,
    "name": "Magic 2015 Core Set",
    "releasedate": "2014-07-18 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "JOU",
    "id": 4655,
    "modern": 1,
    "name": "Journey into Nyx",
    "releasedate": "2014-05-02 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "BNG",
    "id": 4666,
    "modern": 1,
    "name": "Born of the Gods",
    "releasedate": "2014-02-07 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "THS",
    "id": 875,
    "modern": 1,
    "name": "Theros",
    "releasedate": "2013-09-27 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "M14",
    "id": 203,
    "modern": 1,
    "name": "Magic 2014 Core Set",
    "releasedate": "2013-07-19 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "DGM",
    "id": 4685,
    "modern": 1,
    "name": "Dragon's Maze",
    "releasedate": "2013-05-03 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "GTC",
    "id": 4571,
    "modern": 1,
    "name": "Gatecrash",
    "releasedate": "2013-02-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "RTR",
    "id": 872,
    "modern": 1,
    "name": "Return to Ravnica",
    "releasedate": "2012-10-05 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "M13",
    "id": 343,
    "modern": 1,
    "name": "Magic 2013",
    "releasedate": "2012-07-13 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "AVR",
    "id": 868,
    "modern": 1,
    "name": "Avacyn Restored",
    "releasedate": "2012-05-04 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "DKA",
    "id": 4662,
    "modern": 1,
    "name": "Dark Ascension",
    "releasedate": "2012-02-03 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "ISD",
    "id": 867,
    "modern": 1,
    "name": "Innistrad",
    "releasedate": "2011-09-30 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "M12",
    "id": 165,
    "modern": 1,
    "name": "Magic 2012",
    "releasedate": "2011-07-15 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "NPH",
    "id": 671,
    "modern": 1,
    "name": "New Phyrexia",
    "releasedate": "2011-05-13 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "MBS",
    "id": 861,
    "modern": 1,
    "name": "Mirrodin Besieged",
    "releasedate": "2011-02-04 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "SOM",
    "id": 860,
    "modern": 1,
    "name": "Scars of Mirrodin",
    "releasedate": "2010-10-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "M11",
    "id": 164,
    "modern": 1,
    "name": "Magic 2011",
    "releasedate": "2010-07-16 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "ROE",
    "id": 856,
    "modern": 1,
    "name": "Rise of the Eldrazi",
    "releasedate": "2010-04-23 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "WWK",
    "id": 4599,
    "modern": 1,
    "name": "Worldwake",
    "releasedate": "2010-02-05 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "ZEN",
    "id": 852,
    "modern": 1,
    "name": "Zendikar",
    "releasedate": "2009-10-02 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "M10",
    "id": 17,
    "modern": 1,
    "name": "Magic 2010",
    "releasedate": "2009-07-17 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "ARB",
    "id": 4590,
    "modern": 1,
    "name": "Alara Reborn",
    "releasedate": "2009-04-30 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "CON",
    "id": 2644,
    "modern": 1,
    "name": "Conflux",
    "releasedate": "2009-02-06 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "ALA",
    "id": 848,
    "modern": 1,
    "name": "Shards of Alara",
    "releasedate": "2008-10-03 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "EVE",
    "id": 8426,
    "modern": 1,
    "name": "Eventide",
    "releasedate": "2008-07-25 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "SHM",
    "id": 847,
    "modern": 1,
    "name": "Shadowmoor",
    "releasedate": "2008-05-02 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "MOR",
    "id": 8340,
    "modern": 1,
    "name": "Morningtide",
    "releasedate": "2008-02-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "LRW",
    "id": 845,
    "modern": 1,
    "name": "Lorwyn",
    "releasedate": "2007-10-12 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "10E",
    "id": 15,
    "modern": 1,
    "name": "Tenth Edition",
    "releasedate": "2007-07-13 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "FUT",
    "id": 8391,
    "modern": 1,
    "name": "Future Sight",
    "releasedate": "2007-05-04 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "PLC",
    "id": 8423,
    "modern": 1,
    "name": "Planar Chaos",
    "releasedate": "2007-02-02 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "TSP",
    "id": 842,
    "modern": 1,
    "name": "Time Spiral",
    "releasedate": "2006-10-06 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "CSP",
    "id": 5744,
    "modern": 0,
    "name": "Coldsnap",
    "releasedate": "2006-07-21 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "DIS",
    "id": 6451,
    "modern": 1,
    "name": "Dissension",
    "releasedate": "2006-05-05 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "GPT",
    "id": 4657,
    "modern": 1,
    "name": "Guildpact",
    "releasedate": "2006-02-03 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "RAV",
    "id": 161,
    "modern": 1,
    "name": "Ravnica: City of Guilds",
    "releasedate": "2005-10-07 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "9ED",
    "id": 14,
    "modern": 1,
    "name": "Ninth Edition",
    "releasedate": "2005-07-29 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "SOK",
    "id": 6531,
    "modern": 1,
    "name": "Saviors of Kamigawa",
    "releasedate": "2005-06-03 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "BOK",
    "id": 6498,
    "modern": 1,
    "name": "Betrayers of Kamigawa",
    "releasedate": "2005-02-04 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "CHK",
    "id": 837,
    "modern": 1,
    "name": "Champions of Kamigawa",
    "releasedate": "2004-10-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "5DN",
    "id": 3208,
    "modern": 1,
    "name": "Fifth Dawn",
    "releasedate": "2004-06-04 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "DST",
    "id": 727,
    "modern": 1,
    "name": "Darksteel",
    "releasedate": "2004-02-06 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "MRD",
    "id": 836,
    "modern": 1,
    "name": "Mirrodin",
    "releasedate": "2003-10-02 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "8ED",
    "id": 13,
    "modern": 1,
    "name": "Eighth Edition",
    "releasedate": "2003-07-28 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "SCG",
    "id": 6477,
    "modern": 0,
    "name": "Scourge",
    "releasedate": "2003-05-26 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "LGN",
    "id": 2823,
    "modern": 0,
    "name": "Legions",
    "releasedate": "2003-02-03 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "ONS",
    "id": 337,
    "modern": 0,
    "name": "Onslaught",
    "releasedate": "2002-10-07 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "JUD",
    "id": 3015,
    "modern": 0,
    "name": "Judgment",
    "releasedate": "2002-05-27 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "TOR",
    "id": 2191,
    "modern": 0,
    "name": "Torment",
    "releasedate": "2002-02-04 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "OD",
    "id": 832,
    "modern": 0,
    "name": "Odyssey",
    "releasedate": "2001-10-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "AP",
    "id": 4777,
    "modern": 0,
    "name": "Apocalypse",
    "releasedate": "2001-06-04 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "7E",
    "id": 12,
    "modern": 0,
    "name": "Seventh Edition",
    "releasedate": "2001-04-11 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "PS",
    "id": 4811,
    "modern": 0,
    "name": "Planeshift",
    "releasedate": "2001-02-05 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "IN",
    "id": 830,
    "modern": 0,
    "name": "Invasion",
    "releasedate": "2000-10-02 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "PR",
    "id": 8306,
    "modern": 0,
    "name": "Prophecy",
    "releasedate": "2000-06-05 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "NE",
    "id": 8440,
    "modern": 0,
    "name": "Nemesis",
    "releasedate": "2000-02-14 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "MM",
    "id": 402,
    "modern": 0,
    "name": "Mercadian Masques",
    "releasedate": "1999-10-04 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "P3",
    "id": 9,
    "modern": 0,
    "name": "Starter 1999",
    "releasedate": "1999-07-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "CG",
    "id": 4802,
    "modern": 0,
    "name": "Urza's Destiny",
    "releasedate": "1999-06-07 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "PK",
    "id": 822,
    "modern": 0,
    "name": "Portal Three Kingdoms",
    "releasedate": "1999-05-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "6E",
    "id": 8,
    "modern": 0,
    "name": "Classic Sixth Edition",
    "releasedate": "1999-04-21 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "GU",
    "id": 7490,
    "modern": 0,
    "name": "Urza's Legacy",
    "releasedate": "1999-02-15 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "UZ",
    "id": 471,
    "modern": 0,
    "name": "Urza's Saga",
    "releasedate": "1998-10-12 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "EX",
    "id": 7457,
    "modern": 0,
    "name": "Exodus",
    "releasedate": "1998-06-15 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "P2",
    "id": 7,
    "modern": 0,
    "name": "Portal Second Age",
    "releasedate": "1998-06-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "ST",
    "id": 4821,
    "modern": 0,
    "name": "Stronghold",
    "releasedate": "1998-03-02 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "TE",
    "id": 282,
    "modern": 0,
    "name": "Tempest",
    "releasedate": "1997-10-14 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "WL",
    "id": 6482,
    "modern": 0,
    "name": "Weatherlight",
    "releasedate": "1997-06-09 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "PO",
    "id": 83,
    "modern": 0,
    "name": "Portal",
    "releasedate": "1997-05-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "5E",
    "id": 6,
    "modern": 0,
    "name": "Fifth Edition",
    "releasedate": "1997-03-24 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "VI",
    "id": 6454,
    "modern": 0,
    "name": "Visions",
    "releasedate": "1997-02-03 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "MI",
    "id": 466,
    "modern": 0,
    "name": "Mirage",
    "releasedate": "1996-10-08 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "AL",
    "id": 6121,
    "modern": 0,
    "name": "Alliances",
    "releasedate": "1996-06-10 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "HM",
    "id": 5904,
    "modern": 0,
    "name": "Homelands",
    "releasedate": "1995-10-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "IA",
    "id": 280,
    "modern": 0,
    "name": "Ice Age",
    "releasedate": "1995-06-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "4E",
    "id": 5,
    "modern": 0,
    "name": "Fourth Edition",
    "releasedate": "1995-04-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "FE",
    "id": 4701,
    "modern": 0,
    "name": "Fallen Empires",
    "releasedate": "1994-11-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "DK",
    "id": 4253,
    "modern": 0,
    "name": "The Dark",
    "releasedate": "1994-08-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "LE",
    "id": 3465,
    "modern": 0,
    "name": "Legends",
    "releasedate": "1994-06-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "3E",
    "id": 4,
    "modern": 0,
    "name": "Revised Edition",
    "releasedate": "1994-04-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "AQ",
    "id": 3158,
    "modern": 0,
    "name": "Antiquities",
    "releasedate": "1994-03-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "AN",
    "id": 1591,
    "modern": 0,
    "name": "Arabian Nights",
    "releasedate": "1993-12-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "2U",
    "id": 3,
    "modern": 0,
    "name": "Unlimited Edition",
    "releasedate": "1993-12-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "2E",
    "id": 2,
    "modern": 0,
    "name": "Limited Edition Beta",
    "releasedate": "1993-10-01 00:00:00",
    "standard": 0,
  },
  Object {
    "code": "1E",
    "id": 1,
    "modern": 0,
    "name": "Limited Edition Alpha",
    "releasedate": "1993-08-05 00:00:00",
    "standard": 0,
  },
]
`;
